reactivesocket-core/src/main/java/io/reactivesocket/frame/FrameHeaderFlyweight.java

```diff
@@ -79,39 +85,40 @@ public static int computeFrameHeaderLength(final FrameType frameType, int metada
     }
 
     public static int encodeFrameHeader(
-        final MutableDirectBuffer mutableDirectBuffer,
-        final int offset,
-        final int frameLength,
-        final int flags,
-        final FrameType frameType,
-        final int streamId
+            final MutableDirectBuffer mutableDirectBuffer,
+            final int offset,
+            final int frameLength,
+            final int flags,
+            final FrameType frameType,
+            final int streamId
     ) {
         if (INCLUDE_FRAME_LENGTH) {
```

Comment:
Leaving for later, I don't plan to make this better. It seems like a larger task to move this out of framing altogether.